FiveCalls/FiveCalls/MultipleDayPicker.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  MultipleDayPicker.swift
+//  FiveCalls
+//
+//  Created by Christopher Selin on 9/13/23.
+//  Copyright © 2023 5calls. All rights reserved.
+//
+
+import SwiftUI
+
+struct Day: Hashable {
+    let index: Int
+    let name: String
+}
+
+let days = [
+    Day(index: 1, name: "Sun"),
+    Day(index: 2, name: "Mon"),
+    Day(index: 3, name: "Tues"),
+    Day(index: 4, name: "Wed"),
+    Day(index: 5, name: "Thur"),
+    Day(index: 6, name: "Fri"),
+    Day(index: 7, name: "Sat")
+]
+
+struct MultipleDayPicker: View {
+    @Binding var selectedDayIndices: [Int]
+
+    var borderColor: Color { selectedDayIndices.isEmpty ? Color(R.color.red()!) : Color(R.color.darkBlue()!)
+    }
+    
+    var body: some View {
+        HStack(spacing: 0) {
+            ForEach(days, id: \.self) { day in
+                Text(String(day.name))
+                    .foregroundColor(isIndexSelected(day.index) ? Color(R.color.lightBlue()!) : Color(R.color.mediumDarkGray()!))
```

Comment:
does R have a better way to generate `Color` rather than `UIColor`? I wrapped the other ones in statics in `Colors.swift` which at least lets us avoid a bunch of boilerplate color conversions and `!` everywhere its used

Reply:
Looks like there's a better way to do this with SwiftUI (https://medium.com/tbc-engineering/r-swift-swiftui-802acb5560ff). The static approach in Colors.swift just goes around R so we lose compile-time checks. I just verified it works and its as simple as adding `.color` to the end: `R.color.darkBlue.color`